cubedash/_pages.py

```diff
@@ -3,12 +3,12 @@
 from datetime import datetime, timedelta, timezone
 
 import datacube
-import dateutil.parser
 import flask
 import structlog
 from datacube.model import Product, Range
 from datacube.scripts.dataset import build_dataset_info
 from dateutil import tz
+from dateutil.parser._parser import ParserError
```

Comment:
```suggestion
from dateutil.parser import ParserError
```

See https://dateutil.readthedocs.io/en/stable/parser.html#warnings-and-exceptions

Reply:
Fixed.